tests/misc/tconv.nim

```diff
@@ -1,7 +1,4 @@
 discard """
-  nimout:'''
-tconv.nim(81, 15) Warning: enum to enum conversion is now deprecated [User]
-'''
```

Comment:
Why not keep a test for the warning?

Reply:
see https://github.com/nim-lang/Nim/pull/18546#discussion_r673501303 which does exactly that